ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -484,20 +201,41 @@ The list displays all tokens if the *Hide processed* toggle is disabled. Dependi
 - *Process action ID*  
     Action identification number. The ID number is automatically assigned by the system.
 
+### Comments
+
+*Process Orchestration > Processes > Tab OVERVIEW > Select a process > Tab Comments*
+
+![Saved comment](../../Assets/Screenshots/ActindoWorkFlow/Processes/CommentSaved.png "[Saved comment]")
+
+Use the *Comments* tab to add process-specific comments.   
+
+The following functions are available in this view.
+
+- [ADD COMMENT]   
+    Click this button to add a comment in Markdown syntax.    
+    The Add comment view is displayed.
```

Comment:
The *Add comment* window...

Ich finde Add/Edit comment ist ein Fenster, nicht eine Ansicht. Theoretisch sollte man einen separated Header mit "Add comment" Fenster beschreiben, obwohl das Fenster an sich nicht so viel zu beschreiben hat. Vielleicht deutlicher machen, dass CLOSE und SAVE zu Add/Edit comment-Fenster gehören (sie werden hier nicht direkt angezeigt).